src/Resources/app/storefront/src/scss/category-extend.scss

```
@@ -0,0 +1,75 @@
+.category-filter-container {
+  min-width: 290px;
+  label {
+    display: inline-block;
+    width: auto;
+    padding: 0px;
```

I think the IDE already should point out, that a unit value is not required when setting `0` as a value.
```suggestion
    padding: 0;
```

This is done in multiple places in this file. Do this for all occurrences.

---

Done.